### DISCUSSION TITLE
关于BasicVSR++在Vimeo数据集训练的问题

### DISCUSSION THREAD

**Discussion by Kimsure** [open-mmlab/mmediting#1198] [Models]
@ckkelvinchan 您好，非常感谢您的工作！在我对BasicVSR++进行训练复现时，在REDS数据集上可以得到相应结果。但在Vimeo数据集上，我注意到论文中表述的是" Therefore, we initialize the model using the weights trained on REDS when trained on Vimeo-90K. The number of finetune iterations is 300K"，这里是不是说 在Vimeo数据集上训练，开始时直接load在REDS上训好的模型文件，然后训练300k次iterations就可以了？我之前是这么做的 但效果似乎达不到论文水平
期待您的答复！

**Comment by mm-assistant** [comment]
We recommend using English or English & Chinese for issues so that we could have broader discussion.

**Comment by woaicv** [comment]
你好，请问一下basicvsr++你是如何训练的？我没看到git上相关的引导以及训练细节，有没有相关的指令？

**Comment by Kimsure** [comment]
> 你好，请问一下basicvsr++你是如何训练的？我没看到git上相关的引导以及训练细节，有没有相关的指令？

请查阅mmedit教程文档：https://mmediting.readthedocs.io/zh_CN/latest/quick_run.html#id4
有详细指令解释
